Tests/Functional/PhpcrRepositoryTestCase.php

```diff
@@ -11,32 +11,98 @@
 
 namespace Symfony\Cmf\Bundle\ResourceBundle\Tests\Functional;
 
-use Symfony\Cmf\Component\Testing\Functional\BaseTestCase;
-use Doctrine\ODM\PHPCR\Document\Generic;
+use Jackalope\Node;
 
-class PhpcrRepositoryTestCase extends BaseTestCase
+/**
+ * @author Maximilian Berghoff <Maximilian.Berghoff@mayflower.de>
+ */
+abstract class PhpcrRepositoryTestCase extends RepositoryTestCase
 {
-    protected $dm;
+    /**
+     * @dataProvider provideGet
+     */
+    public function testRepositoryGet($path, $expectedName)
+    {
+        $res = $this->getRepository()->get($path);
+        $this->assertNotNull($res);
+        $payload = $res->getPayload();
+
+        $this->assertEquals(
+            $expectedName,
+            ($payload instanceof Node ? $payload->getName() : $payload->getNodeName())
```

Comment:
`s/Node/NodeInterface` - PHPCR is not Jackalope. PHPCR should always be type hinted with the PHPCR interfaces, i.e. `PHPCR\NodeInterface`.